open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -682,28 +682,28 @@ def schedule_pre_allocate_expr_job(self):
                                     method="check_hackathon_for_pre_allocate_expr",
                                     id="check_hackathon_for_pre_allocate_expr",
                                     next_run_time=next_run_time,
-                                    minutes=10)
+                                    minutes=20)
 
     def check_hackathon_for_pre_allocate_expr(self):
         """Check all hackathon for pre-allocate
 
         Add an interval job for hackathon if it's pre-allocate is enabled.
         Otherwise try to remove the schedule job
         """
-        # todo fix pre-allocate
-        # hackathon_list = self.db.find_all_objects(Hackathon)
-        hackathon_list = []
+        hackathon_list = Hackathon.objects()
+        index = 0
         for hack in hackathon_list:
             job_id = "pre_allocate_expr_" + str(hack.id)
             is_job_exists = self.scheduler.has_job(job_id)
-            if hack.is_pre_allocate_enabled():
+            hack.save()
+            if hack.config['pre_allocate_enabled']:
```

Comment:
The operation 'config.pre_allocate_enabled` is not available. `config.get("pre_allocate_enabled")` is ok